presto-docs/src/main/sphinx/release/release-0.235.rst

```diff
@@ -0,0 +1,64 @@
+=============
+Release 0.235
+=============
+
+General Changes
+_______________
+* Fix named queries where its output does not match its column list (:issue:`14333`).
+* Fix error in analysis phase.
```

Comment:
Need some clarification / short description what the error is. Does it cause query failure? Avoid "analysis phase" if possible since user does not need to understand that, as user sees "query failure" or "incorrect result", etc.

Reply:
Let's just remove this one. It's very hard to describe. And it's not necessarily tied to certain user facing features. Or maybe "fix query failure with NullPointerException during analysis. :issue:`14340`".

Reply:
I'll just remove it.